recipes/r-doubletfinder/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: r-doubletfinder
+  version: 1.0.0
+
+source:
+  url: https://github.com/chris-mcginnis-ucsf/DoubletFinder/archive/03e9f37f891ef76a23cc55ea69f940c536ae8f9f.zip
```

Comment:
Any chance you could ask the author to make a release/tag ?

Reply:
I could, but would probably take a while. 
Would it be an acceptable intermediary  solution to have it as a 0.0.1 version until we have a proper tag?